deploy/08-ca-daemonset.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: registry-ca-hostmapper
+  namespace: openshift-image-registry
+spec:
+  selector:
+    matchLabels:
+      name: registry-ca-hostmapper
+  template:
+    metadata:
+      labels:
+        name: registry-ca-hostmapper
+    spec:      
+      tolerations:
+      - effect: NoSchedule
+        key: node-role.kubernetes.io/master
+        operator: Exists
+      serviceAccountName: registry-ca-hostmapper
+      containers:
+      - name: registry-ca-hostmapper
+        securityContext:
+          privileged: true
+        image: docker.io/centos:7
+        command: 
+        - "/bin/sh"
+        args:
+        - "-c"
+        - "mkdir /etc/docker/certs.d/image-registry.openshift-image-registry.svc.cluster.local:5000; cp /tmp/serviceca/service-ca.crt /etc/docker/certs.d/image-registry.openshift-image-registry.svc.cluster.local:5000; sleep 60"
```

Comment:
what is the sleep for?

Reply:
so the pod dies every minute and restarts, which will cause it to pick up any change to the CA in the configmap and re-copy it.